telegram/_stars.py

```diff
@@ -420,7 +449,17 @@ def __init__(
         self, transactions: Sequence[StarTransaction], *, api_kwargs: Optional[JSONDict] = None
     ):
         super().__init__(api_kwargs=api_kwargs)
-        self.transactions: Sequence[StarTransaction] = transactions
+        self.transactions: Tuple[StarTransaction, ...] = parse_sequence_arg(transactions)
 
         self._id_attrs = (self.transactions,)
         self._freeze()
+
+    @classmethod
+    def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["StarTransactions"]:
+        data = cls._parse_data(data)
+
+        if not data:
+            return None
```

Comment:
☂️ 🙂

Reply:
we did the same for basically every other class.. hmm..